src/DataSource.ts

```diff
@@ -72,7 +87,7 @@ export class DataSource extends DataSourceWithBackend<TimestreamQuery, Timestrea
       database: templateSrv.replace(query.database || '', scopedVars),
       table: templateSrv.replace(query.table || '', scopedVars),
       measure: templateSrv.replace(query.measure || '', scopedVars),
-      rawQuery: templateSrv.replace(query.rawQuery, queryScopedVars),
+      rawQuery: templateSrv.replace(query.rawQuery, queryScopedVars, this.interpolateVariable),
```

Comment:
is there a way of testing this?

Reply:
I was discouraged by the fact that there were no test at all for this class but yes, in this case it can be useful for the future.